components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -680,6 +684,8 @@ private void populateRequiredAttributesForIdentityProviderList(ResultSet resultS
                     }
                 }
             }
+        } catch (IdentityProviderManagementClientException e) {
+            throw e;
```

Comment:
no need to catch and throw

Reply:
If not catch as IdentityProviderManagementException and throw IdentityProviderManagementServerException https://github.com/wso2/carbon-identity-framework/pull/3976/files/bf3c1b4257fca6bff8fc8476df596386891c7dda#diff-6af8c5bc4fe450a0e7aaa28ea162b84d3e862c57f60cfe438e8fd0030539ce72R689